modal/modal.css

```diff
@@ -0,0 +1,63 @@
+.modal {
+  width: 55%;
+  text-align: center;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background-color: #bb453073;
```

Comment:
use color variables for colors, it's a good practice and very useful

Reply:
Done